Update `org.bouncycastle:bcprov-jdk15on` dependency

HI! Since our project uses your library and both [web3j](https://github.com/web3j/web3j) and [Wallet Connect V2](https://github.com/WalletConnect/WalletConnectKotlinV2) when building the project we are getting errors connected with unsupported `Bouncy Castle` versions:
```
Caused by: com.android.builder.merge.DuplicateRelativeFileException: 2 files found with path 'org/bouncycastle/x509/CertPathReviewerMessages_de.properties'.
```

Everything is working fine when I exclude `Bouncy Castle` from Solana library
```
    implementation("com.github.metaplex-foundation:SolanaKT:2.0.1") {
        exclude(group = "org.bouncycastle", module = "bcprov-jdk15to18")
    }
```

But since both `web3j` and `Wallet Connect` uses the same version of `org.bouncycastle` which is `1.7.0` it will also be possible to update it in the `SolanaKT` library from `1.6.8` to `1.7.0`

--- a/solana/build.gradle
+++ b/solana/build.gradle
@@ -52,7 +52,7 @@ android {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     implementation 'com.squareup.okhttp3:okhttp:4.9.0'
-    implementation 'org.bitcoinj:bitcoinj-core:0.15.10'
+    implementation 'org.bitcoinj:bitcoinj-core:0.16.2'
     implementation 'net.i2p.crypto:eddsa:0.3.0'
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'

